scripts/UpdateDependencies.ps1

```diff
@@ -17,36 +18,125 @@ Set-StrictMode -Version 1
 $depsPath = Resolve-Path "$PSScriptRoot/../build/dependencies.props"
 [xml] $dependencies = LoadXml $depsPath
 
-if ($BuildXml -like 'http*') {
-    $url = $BuildXml
-    New-Item -Type Directory "$PSScriptRoot/../obj/" -ErrorAction Ignore
-    $localXml = "$PSScriptRoot/../obj/build.xml"
-    Write-Verbose "Downloading from $url to $BuildXml"
-    Invoke-WebRequest -OutFile $localXml $url
-}
+$variables = @{}
 
-[xml] $remoteDeps = LoadXml $localXml
-$count = 0
+if ($UseCoreFx) {
+    if ($BuildXml) {
+        Write-Warning "$BuildXml parameter will not be used when the UseCoreFx switch is passed."
```

Comment:
Seems like a code smell. Might be cleaner to have separate script files, and extract the shared code into functions that you import via a .psm1 file